### ISSUE TITLE
Integrate Homepage

### ISSUE THREAD

**Issue by kater1** [ustwo/ustwo.com-frontend#146]

**Comment by matthewedwards12** [comment]
Homepage needs integration so that editors can amend and add content. 3 sections may not – eventually – be enough, although Related Content might make this a non-issue.

**Comment by kater1** [comment]
Elementary working on this for when we prioritise this integration

**Comment by collingo** [comment]
Just to add the notes Phil sent to Andy about the backend work relating to this...

1) It needs a hero component like the other pages,

2) A new component is needed, Feature Block, that includes:

    a) Heading

     b) Description - rich text so we can add links into the text?

    c) Much like Page colour scheme on other pages, we need a 'Text' colour and 'Background' colour. Could these be dropdowns? e.g. for 'Text' we would need Black or White. The 'Background' dropdown could be a list of the ustwo colour palette (also content managed somewhere).

    d) Image - For the time being we will definitely just deal with one image but we need to be able to pick between a jpg or a png. The jpg is being used on smaller screens (and when the gradient 'scroll magic' component isn't activated). Is there any chance a png can be uploaded and then combined with the 'background' colour that spits out a jpg?! If this is a big task then two fields to upload one png and one jpg will be fine for now.

3) Page colour scheme at the bottom like other pages. We would need this to allow us to set the background colour of the hero. Incidentally, this is something that would be nice to have in the hero component itself (again with a dropdown of ustwo colours). However, we don't want to create more work that could delay the main job at hand so we thought for the time being we can just use this existing colour scheme component.

**Comment by collingo** [comment]
Need to establish the data contract with Andy before we can begin integration who is no in the office today so putting this on hold for now.

**Comment by collingo** [comment]
Elementary still have some work to do in order to complete this after we reprioritised this story half way through. Andy is picking up today and will update us when it's ready to roll

**Comment by collingo** [comment]
Blocked by api fixes and asset generation
 - `feature_block` text colour needs to be offWhite and nonBlack rather than the ustwo palette
 - `feature_block` colour selections are ignored for the second and above blocks
 - new assets needed for the `feature_block` images and a decision on whether to bake in the svg widgets needs to be made

**Comment by collingo** [comment]
Unblocked

**Comment by collingo** [comment]
Unblocked - just need to finalise assets before pushing to prod
